### PR TITLE
Build page on `/azure/sql`

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build page on `/azure/sql`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please compare page with [copy doc](https://docs.google.com/document/d/16QHk3drMIRLeeEdYurIQieo1W93SygetyWdTT4184pE/edit#)
- Demo at https://ubuntu-com-11412.demos.haus/azure/sql

## Issue / Card

Fixes [#4993](https://github.com/canonical-web-and-design/web-squad/issues/4993)

## Screenshots

SQL Server on Ubuntu Pro for Azure | Ubuntu (2022-03-29 13-21-05)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @minkyngkm, I have one suggestion. As we don't have images of matching sizes/type for the "Get more out of SQL Server on Ubuntu Pro for Azure" section, I would propose using the strip below, with the buttons added of course.

image

Thanks

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for reviewing! I've updated that section. Hope it looks better now! [Demo](https://ubuntu-com-11412.demos.haus/azure/sql)

**Comment by spencerbygraves** [comment]
Thanks @minkyngkm, I've made a slight mistake! Could we use `p-strip--square-suru` (I think that's the right one) as the angle meeting the footer doesn't work.

Also, you're missing the buttons from the copy doc.

So from this...

image

To this...

image




Sorry!

Thanks

**Comment by minkyngkm** [comment]
@spencerbygraves Thanks Spencer! Does it look better now? :)
image

**Comment by spencerbygraves** [comment]
@minkyngkm looks good, thanks!

File: templates/azure/sql.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        </div>
+      </li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>24/7 supported SQL Server on Ubuntu Pro. Backed by Canonical and Microsoft.</h2>
+    </div>
```
Comment (sowasred2012) [inline templates/azure/sql.html:118]: It looks a little odd wrapped onto three lines, perhaps we could break it into two lines like this:
```suggestion
    <div class="col-12">
      <h2>24/7 supported SQL Server on Ubuntu Pro.<br>Backed by Canonical and Microsoft.</h2>
    </div>
```

**Review by sowasred2012 — APPROVED** [review]
LGTM! Just one suggestion, then good to merge

**Pull request merged**